service/src/globals/tokio_handle.rs

```diff
@@ -59,6 +59,25 @@ impl GetTokioHandle for GlobalTokioHandle {
 	}
 }
 
+/// Implementation for a scoped Tokio handle
+///
+///
+pub struct ScopedTokioHandle {
+	tokio_runtime: tokio::runtime::Runtime,
+}
```

Comment:
Added a new scoped tokio handle for use in tests. As opposed to the global tokio handle, which is initialized once at startup of the application.